x-pack/plugins/security_solution/common/utils/field_formatters.test.ts

```diff
@@ -162,14 +163,14 @@ describe('Events Details Helpers', () => {
       },
       {
         category: 'signal',
-        field: 'signal.status',
+        field: ALERT_STATUS,
```

Comment:
Unclear to me if we're using `ALERT_STATUS` or `ALERT_WORKFLOW_STATUS` (`ALERT_STATUS` isn't actually in the spreadsheet). Open question...

Reply:
pretty sure it's going to be ALERT_WORKFLOW_STATUS, we'll update these